frontend/src/components/popups/Signup.jsx

```diff
@@ -25,9 +26,17 @@ function Signup() {
                 type: 'OPEN_POPUP',
                 payload: 'login'
             })
-            console.log("Registration successful:", response.data);
+            menuDispatch({
+                type: 'SHOW_ALERT',
+                payload: {type: 'success',
+                          message: 'Registration Successful!' }
+            })
         } catch (error) {
-            console.log("Registration failed:", error);
+            menuDispatch({
+                type: 'SHOW_ALERT',
+                payload: {type: 'error',
+                          message: error.response.data.message }
```

Comment:
this is definitely nit-picky but I don't really like this type of indenting, for object I think it should always be like this

```
            menuDispatch({
                type: 'SHOW_ALERT',
                payload: {
                    type: 'error',
                    message: error.response.data.message
                }
```

Reply:
Okay, this is helpful because whenever I make an object, I don't have a preferred way to format it, so its good to keep it consistent.